README.md

```diff
@@ -217,27 +217,9 @@ The important configuration is listed as following.
 |rss.server.flush.cold.storage.threshold.size|64M| The threshold of data size for LOACALFILE and HDFS if MEMORY_LOCALFILE_HDFS is used|
 
 
-### Spark Client
+### Shuffle Client
```

Comment:
Why is Shuffle Client?

Reply:
The linked page contains both Spark and MR clients